Kickstarter.xcodeproj/project.pbxproj

```diff
@@ -9239,6 +9239,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
+				MARKETING_VERSION = 5.8.0;
```

Comment:
Do not introduce uneccessary changes here. If it previously was not there, no need to add it. Only need modifications with this. Ie. 5.7.0 -> 5.8.0